packages/ember-runtime/lib/computed/computed_macros.js

```diff
@@ -80,8 +80,8 @@ function generateComputedWithPredicate(name, predicate) {
   @method empty
   @for Ember.computed
   @param {String} dependentKey
-  @return {Ember.ComputedProperty} computed property which negate
-  the original value for property
+  @return {Ember.ComputedProperty} computed property which returns true if
+  original value for property is null, an empty string, empty array, or empty function.
```

Comment:
Can you also mention what it returns when the underlying value is not empty? (I presume false, but it would be nice to say it explicitly)?